specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,375 @@
+        "properties": {
+          "createdBy": {
+            "$ref": "#/components/schemas/CreatedBy"
+          },
+          "penaltyIdentifier": {
+            "$ref": "#/components/schemas/PenaltyIdentifier"
+          },
+          "reasons": {
+            "$ref": "#/components/schemas/Reason"
+          }
+        }
+      },
+      "Attachment": {
+        "required": [
+          "contentType",
+          "id",
+          "name",
+          "size"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "contentType": {
+            "type": "string"
+          },
+          "size": {
+            "minimum": 1,
+            "type": "integer",
+            "format": "int32"
+          },
+          "url": {
+            "type": "string"
+          }
+        }
+      },
+      "CreatedBy": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "emailAddress": {
+            "type": "string"
+          }
+        }
+      },
+      "OtherReason": {
+        "required": [
+          "description",
+          "title"
+        ],
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "attachments": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Attachment"
+            }
+          }
+        }
+      },
+      "IllnessReason": {
+        "required": [
+          "illPerson",
+          "illnessStartDate",
+          "continuedIllness",
+          "illnessEndDate",
+          "illnessImpactFurtherInformation",
+          "attachments"
+        ],
+        "type": "object",
+        "properties": {
+          "illPerson": {
+            "type": "string"
+          },
+          "illnessStartDate": {
+            "type": "string"
+          },
+          "continuedIllness": {
+            "type": "boolean"
+          },
+          "illnessEndDate": {
+            "type": "string"
+          },
+          "illnessImpactFurtherInformation": {
+            "type": "string"
+          },
+          "attachments": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Attachment"
+            }
+          }
+        }
+      },
+      "PenaltyIdentifier": {
+        "required": [
+          "penaltyReference"
+        ],
+        "type": "object",
+        "properties": {
+          "companyNumber": {
+            "type": "string"
+          },
+          "penaltyReference": {
+            "pattern": "([A-Z0-9/]{8,14})",
+            "type": "string"
+          }
+        }
+      },
+      "Reason": {
+        "required": [
+          "illnessReason",
+          "other"
```

Comment:
Both illness and other reasons won't be required. The user is only required to submit one of them.